example/cumulus-tf/ecs_hello_world_workflow.tf

```diff
@@ -44,31 +39,14 @@ module "hello_world_service" {
   }
 }
 
-resource "aws_cloudwatch_metric_alarm" "ecs_hello_world_task_count_high" {
```

Comment:
So we can't include these alarms because we no longer have any configuration to know whether a task count is high or low? Thus, there won't be any more ECS alarms in our Cloudwatch dashboard?

Reply:
This alarm became relatively useless as soon as we added autoscaling. It sets the threshold to 1 and alarms if the count goes above that, which should not be an alarming situation if autoscaling has triggered it to scale up to more than 1. Also, it's hard-coded to 1, which may not be the default case depending on what people have their `desired_count` set to.

Reply:
Makes sense. Just didn't know if having these ECS alarms was part of an objective or anything, but regardless we can't maintain the alarm without knowing their custom settings